conf/app.routes

```diff
@@ -12,5 +12,12 @@ POST          /fm/registration/:id
 
 
 POST          /subscription/create-subscription                                     uk.gov.hmrc.pillar2.controllers.SubscriptionController.createSubscription
-GET           /subscription/read-subscription/:id/:plrReference                     uk.gov.hmrc.pillar2.controllers.SubscriptionController.readSubscription(id: String, plrReference: String)
-PUT           /subscription/amend-subscription                                      uk.gov.hmrc.pillar2.controllers.SubscriptionController.amendSubscription
+GET           /subscription/read-subscription/:id/:plrReference                     uk.gov.hmrc.pillar2.controllers.SubscriptionController.readAndCacheSubscription(id: String, plrReference: String)
```

Comment:
can the routes be renamed to avoid using verbs?

Reply:
What value do we get from it? its just for our own internal use

Reply:
It is internal but it makes it more readable for anyone who needs to use the backend. Not major though.